src/sagemaker/model.py

```diff
@@ -345,6 +375,16 @@ def compile(
                 model. Allowed values: 'mxnet', 'tensorflow', 'keras', 'pytorch',
                 'onnx', 'xgboost'
             framework_version (str):
+            target_platform_os (str): Target Platform OS. Allowed values: 'LINUX', 'ANDROID'.
+                It can be used instead of target_instance_family.
+            target_platform_arch (str): Target Platform Architecture. Allowed values: 'X86_64',
+                'X86', 'ARM64', 'ARM_EABIHF', 'ARM_EABI'.
+                It can be used instead of target_instance_family.
+            target_platform_accelerator (str): Target Platform Accelerator. Allowed values:
+                'INTEL_GRAPHICS', 'MALI', 'NVIDIA'. Optional.
```

Comment:
can you instead link to docs that list the allowed versions? I assume they might change in the future, and if that happens, this docstring will inevitably be forgotten.

Reply:
The API reference is updated.
OutputConfig API
TargetPlatform API

Reply:
I left just one example value and added link to allowed values.